stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/handlers/OnBackPressedHandler.kt

```diff
@@ -25,6 +26,8 @@ public fun SystemBackPressedHandler(
     isEnabled: Boolean,
     onBackPressed: () -> Unit,
 ) {
+    if (LocalInspectionMode.current) return
+
```

Comment:
Disable the functionality when rendering component previews to avoid an error.

Reply:
Nicely done - really good improvement that'll let us build and show more complex previews!